onnxruntime/core/framework/op_kernel.cc

```diff
@@ -9,6 +9,22 @@
 using namespace ::onnxruntime::common;
 namespace onnxruntime {
 
+std::unique_ptr<OpKernelInfo> CopyOpKernelInfo(const OpKernelInfo& info) {
+  return onnxruntime::make_unique<OpKernelInfo>(info);
+}
+
```

Comment:
Not sure this adds much value as seeing the make_unique instead of a call to CopyOpKernelInfo may be more readable.

Reply:
It's a function I can call across a DLL boundary. Can't do make_unique on the shared side, and this function is used in the shared header so it has to work in both cases.